packages/api/src/external/carequality/command/cq-directory/get-cq-directory-entries.ts

```diff
@@ -0,0 +1,69 @@
+import { QueryTypes } from "sequelize";
+import { Pagination, sortForPagination } from "../../../../command/pagination";
+import { CQDirectoryEntry2 } from "../../cq-directory";
+import { HIEDirectoryEntryViewModel } from "../../models/hie-directory-view";
+
+export async function getHieDirectoryEntriesByFilter(
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  { filter, pagination }: { filter: string; pagination: Pagination }
+): Promise<CQDirectoryEntry2[]> {
+  const sequelize = HIEDirectoryEntryViewModel.sequelize;
+  if (!sequelize) throw new Error("Sequelize not found");
+
+  const querySelect = `SELECT * FROM ${HIEDirectoryEntryViewModel.tableName} WHERE 1 = 1 `;
+
+  const queryFTS =
+    querySelect +
+    (filter
+      ? ` AND (search_criteria @@ websearch_to_tsquery('english', :filter) OR id = :filter)`
```

Comment:
Im not too familiar with this query - will this filter across all columns?

Reply:
`search_criteria` is a column I defined in our migration that is a simple string concatenation of all the human readable fields on a row someone might want to search against (think the name of the org, of the root org, the oid, the state and the zip). I actually need to remove the `address_line` and `city` which are also in the `search_criteria` but probably shouldn't be searchable if we're not returning those values.